controller.py

```diff
@@ -66,8 +69,39 @@ def start_new_game():
   (and continuation) of a game of Bites.
   """
   players = prepare_list_of_players()
-  play_bites = Bites(ANTS, TOKENS_FOR_TRAIL, players)
+  anthill_order = choose_anthill_rule()
+  play_bites = Bites(ANTS, TOKENS_FOR_TRAIL, players, anthill_order)
   play_bites.play_full_game()
 
+def choose_anthill_rule(anthill_card_dict=ANTHILL_CARD_DICT):
+  """Allows the user to choose the anthill rule for this game
+
+  Select one of the available options or allow the game to select one at random.
+
+  Parameters
+  ----------
+  anthill_card_dict : (dict)
+    A dictionary containing the available options of anthill rule choice.
+    Referred to as 'cards' to reflect the real Bites game.
+    Keys are names of rules as strings
+    Values are the implementation of those rules as lists.
+
+  Returns
+  -------
+  card_choice : (string)
+    The string value from the keys of anthill_card_dict that has been selected
+  """
+  anthill_card_allowed_choices = list(anthill_card_dict.keys())
+  show_allowed_choices_from_list(anthill_card_allowed_choices)
+  print("Or enter 'random' to select one of the above options")
+  card_choice = input("Please enter your choice of anthill card: ")
```

Comment:
What about the scenario where you want to choose random but accidentally make a typo, e.g. "Random" or "raandom". Has this scenario been accounted for? I've not seen it in the tests.

Reply:
I think that if they enter 'raandom' etc. then the IF is false, the ELIF is also false so the method continues around the loop. I didn't think I'd need a test explicitly for that because it is covered in the logic already.